src/ots.cc

```diff
@@ -630,9 +631,11 @@ bool ProcessGeneric(ots::FontFile *header,
     if (tables[i].uncompressed_length > tables[i].length) {
       // We'll probably be decompressing this table.
 
-      // disallow all tables which uncompress to > 30 MB
-      if (tables[i].uncompressed_length > 30 * 1024 * 1024) {
-        return OTS_FAILURE_MSG_TAG("uncompressed length exceeds 30MB", tables[i].tag);
+      // disallow all tables which uncompress to > OTS_MAX_DECOMP_TABLE_SIZE
+      if (tables[i].uncompressed_length > OTS_MAX_DECOMP_TABLE_SIZE) {
+        return OTS_FAILURE_MSG_HDR("%c%c%c%c: uncompressed table length exceeds %gMB",
```

Comment:
While at it, please update the message to say “decompressed” like the others.

Reply:
done in https://github.com/khaledhosny/ots/pull/229/commits/f21ab4ef2ff0d7c90ed2282341098b7354cbe5a4